payments-core/api/payments-core.api

```diff
@@ -3567,10 +3567,12 @@ public final class com/stripe/android/model/PaymentMethod$Card : com/stripe/andr
 	public final field fingerprint Ljava/lang/String;
 	public final field funding Ljava/lang/String;
 	public final field last4 Ljava/lang/String;
+	public final field networks Lcom/stripe/android/model/PaymentMethod$Card$Networks;
```

Comment:
Should this go into the release notes?

Reply:
I think so, is this work being done on iOS?

Reply:
Yes, let's add to the CHANGEDOC.

Reply:
@jameswoo-stripe It’s already public in the iOS repo.